plugins/ScheduledReports/tests/Integration/ApiTest.php

```diff
@@ -112,8 +120,18 @@ public function getAttachments($report, $processedReports, $prettyDate) {
             $reportIds, [ScheduledReports::DISPLAY_FORMAT_PARAMETER => ScheduledReports::DISPLAY_FORMAT_TABLES_ONLY]);
 
         $eventCalledWith = [];
-        Piwik::addAction(APIScheduledReports::SEND_REPORT_EVENT, function (&$reportType, $report, $contents, $filename, $prettyDate, $reportSubject,
-                                                                           $reportTitle, $additionalFiles, $period, $force) {
+        Piwik::addAction(APIScheduledReports::SEND_REPORT_EVENT, function (
+            &$reportType,
+            $report,
+            $contents,
+            $filename,
+            $prettyDate,
+            $reportSubject,
+            $reportTitle,
+            $additionalFiles,
+            $period,
+            $force
+) {
```

Comment:
```suggestion
        ) {
```

Interestingly zero spaces are accepted, but, if there is at least one space, the autofix goes to 8 space indentation :shrug:

Reply:
Hm... Maybe that would be fixed by another rule at some point